src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
```

Comment:
I would recommend you to order `import` statements, as mentioned in the [guide](https://mate-academy.github.io/style-guides/java/java.html#s3.3.3-import-ordering-and-spacing)

Reply:
Based on the provided guide, my code correctly follows rules 3.3.1, 3.3.2, and 3.3.3. I don't have any static imports or static nested classes, so rule 3.3.4 also doesn't apply. My import statements are properly organized according to the specified guidelines. And mvn checkstyle doesn't say me anything.. Can you specify what I have to change there?